micrometer-core/src/main/java/io/micrometer/core/instrument/binder/grpc/GrpcObservationContextInfo.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2022 the original author or authors.
```

Comment:
Was this really reviewed by a human before opening the PR?
These mistakes are easy to make even for humans (or miss them during reviews) but it does not help me trusting the PR more.